lib/plugins/aws/package/compile/events/apiGateway/lib/hack/updateStage.js

```diff
@@ -290,7 +314,11 @@ function removeLogGroup() {
   const stage = this.options.stage;
   const logGroupName = `/aws/api-gateway/${service}-${stage}`;
 
-  const logs = provider.logs && provider.logs.restApi;
+  let logs = provider.logs && provider.logs.restApi;
```

Comment:
I think it'll be nicer now to name this var `accessLogging`, also maybe we can rename function `removeLogGroup` into `removeAccessLoggingLogGroup`.

For a while I was confused that we're removing log group which also takes execution logs

Reply:
Yes, this stems from the way the yaml settings were originally laid out not really taking into consideration the difference between the two different types of logging and just turning both types on with all options enabled when logging was enabled.

AWS itself creates the log group (API-Gateway-Execution-Logs_*) for execution logging so we don't have control over the naming. Since it's usually used to collect debug info during short periods for troubleshooting I don't think there should be anything in serverless to automatically delete it so I didn't add anything for that.

I renamed the method accordingly so it's no longer confusing, and also changed the variable name since it makes the purpose clearer when reading the code.

Code is ready for review again once CI passes.

Reply:
Great thanks for explanations